gcp-metric-alarm/bool_alarm.tf

```diff
@@ -0,0 +1,26 @@
+resource "google_monitoring_alert_policy" "bool" {
+  count        = var.alarm_configuration.value_type == "BOOL" ? 1 : 0
+  display_name = local.alarm_unique_id
+  combiner     = "OR"
+  conditions {
+    display_name = "${var.md_metadata.name_prefix} BOOL"
```

Comment:
Why aren't we using the var.display_name here? If a package has two bool alarms, they'd have the same name!

Reply:
Thanks for that, you're totally right. I updated this to be the display name variable with no extra interpolation so it matches the MD UI. I also added some notes around the other... `display_name` attribute that needs to be project-unique for this resource to help future users and future me.